Jenkinsfile

```diff
@@ -57,9 +56,18 @@ node {
     }
     echo "Building and testing ${newPkgVersion}"
 
+    sh "docker build -t hypothesis-client-tests ."
+    nodeEnv = docker.image("hypothesis-client-tests")
+
+    stage('Setup') {
+      nodeEnv.inside("-e HOME=${workspace}") {
+        sh "yarn install"
+      }
+    }
+
     stage('Test') {
         nodeEnv.inside("-e HOME=${workspace}") {
-            sh 'make test'
```

Comment:
Instead of running only tests in Jenkins, we now run all CI steps. This helps to ensure that all checks have been run on the master branch before it is deployed to prod. `checkformatting` is not particularly important in this context, but it is also pretty quick to run, so I've included it for consistency.